### PR TITLE
Display estimations on Audience form

### PR DESCRIPTION
**Pull request by coderberry**

### CODE REVIEW

File: assets/src/css/app.scss [hunk 1]
```diff
@@ -6,4 +6,4 @@
 @import "~sweetalert2/src/sweetalert2";
 @import "./selectize.bootstrap4";
 @import "./theme/theme";
-@import "./codefund-framework/codefund-framework";
+@import "./codefund-framework/codefund-framework";
```
Comment (StareIntoTheBeard) [inline assets/src/css/app.scss]: no newline at end of file

File: assets/src/css/codefund-framework/_css-loader.scss [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+.cssload-loader {
+  animation-name: cssload-loader;
+  -o-animation-name: cssload-loader;
+  -ms-animation-name: cssload-loader;
+  -webkit-animation-name: cssload-loader;
+  -moz-animation-name: cssload-loader;
+}
+
+@keyframes cssload-loader {
+  from {
```
Comment (StareIntoTheBeard) [inline assets/src/css/codefund-framework/_css-loader.scss]: can we put these from/to styles in a class and use them in these functions

File: assets/src/css/codefund-framework/codefund-framework.scss [hunk 3]
```diff
@@ -1,2 +1,3 @@
 @import "./custom";
-@import "./form";
\ No newline at end of file
+@import "./form";
+@import "./css-loader";
```
Comment (StareIntoTheBeard) [inline assets/src/css/codefund-framework/codefund-framework.scss]: no new line at end of file

File: assets/src/css/codefund-framework/_css-loader.scss [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+@-moz-keyframes cssload-loader {
+  from {
+    -moz-transform: scale(0);
+    opacity: 1;
+  }
+  to {
+    -moz-transform: scale(1);
+    opacity: 0;
+  }
+}
```
Comment (StareIntoTheBeard) [inline assets/src/css/codefund-framework/_css-loader.scss]: no new line at end of file

File: lib/code_fund_web/templates/coherence/invitation/new.html.eex [hunk 5]
```diff
@@ -1,42 +1,45 @@
...
+          <%= link dgettext("coherence", "Cancel"), to: user_path(@conn, :index), class: "btn" %>
+          <%= if invitation = @conn.assigns[:invitation] do %>
+            <%= link dgettext("coherence", "Resend Invitation!"), to: invitation_path(@conn, :resend, invitation.id), class: "btn" %>
           <% end %>
         </div>
-      </div>
+      <% end %>
     </div>
   </div>
 </div>
```
Comment (StareIntoTheBeard) [inline lib/code_fund_web/templates/coherence/invitation/new.html.eex]: no new line at end of file

File: lib/code_fund_web/templates/user/audience/form.html.eex [hunk 6]
```diff
@@ -1,23 +1,89 @@
...
+        Potential Monthly Audience
+      </div>
+      <div class="card-body d-none" data-target="audience-form.potentialAudienceWithData">
+        <h6 class="mb-0" style="font-size: 13px;">Impressions</h6>
+        <p class="mt-0 mb-1" style="font-size: 11px;">
+          <span data-target="audience-form.estimatedImpressionsLow">0</span>
+          -
+          <span data-target="audience-form.estimatedImpressionsHigh">0</span>
+        </p>
+        <div class="progress" style="height: 10px;">
```
Comment (StareIntoTheBeard) [inline lib/code_fund_web/templates/user/audience/form.html.eex]: are inline styles necessary here?

File: assets/src/css/codefund-framework/_css-loader.scss (hunk 2 above)
Comment (coderberry) [reply assets/src/css/codefund-framework/_css-loader.scss]: I just copied/pasted from another site. I'd rather not refactor unless you insist.

File: lib/code_fund_web/templates/user/audience/form.html.eex (hunk 6 above)
Comment (coderberry) [reply lib/code_fund_web/templates/user/audience/form.html.eex]: Oops.. fixed

File: assets/src/css/codefund-framework/_css-loader.scss (hunk 2 above)
Comment (StareIntoTheBeard) [reply assets/src/css/codefund-framework/_css-loader.scss]: eh not super critical

**Pull request merged**